packages/contracts/src/dollar/facets/BondingCurveFacet.sol

```diff
@@ -46,7 +46,7 @@ contract BondingCurveFacet is Modifiers, IBondingCurve {
         return LibBondingCurve.getShare(_recipient);
     }
 
-    function withdraw(uint256 _amount) external whenNotPaused {
+    function withdraw(uint256 _amount) external onlyAdmin whenNotPaused {
```

Comment:
can you clarify why it is admin only ?

Reply:
I assumed only the admin would have the right and permission to withdraw to the treasury